kernel/byterun/coq_interp.c

```diff
@@ -834,7 +833,7 @@ value coq_interprete
         *--sp=accu;
         Coq_alloc_small(accu, nfuncs * 3 + nvars, Closure_tag);
         Field(accu, nfuncs * 3 + nvars - 1) = *sp++;
-        p = &Field(accu, 0);
+        volatile value *p = &Field(accu, 0);
```

Comment:
Forcing the compiler to generate worse code just for the sake of silencing a warning does not make me too happy. The following should achieve the same goal:
```suggestion
        p = (value *) &Field(accu, 0);
```